ui/src/shared/actions/notifications.js

```diff
@@ -1,31 +1,13 @@
-export function publishNotification(type, message, options = {once: false}) {
-  // this validator is purely for development purposes. It might make sense to move this to a middleware.
-  const validTypes = ['error', 'success', 'warning']
-  if (!validTypes.includes(type) || message === undefined) {
-    console.error('handleNotification must have a valid type and text') // eslint-disable-line no-console
-  }
-
-  return {
-    type: 'NOTIFICATION_RECEIVED',
-    payload: {
-      type,
-      message,
-      once: options.once,
-    },
-  }
-}
-
-export function dismissNotification(type) {
+export function publishNotification(notification) {
```

Comment:
could you update this to `const` syntax?

Reply:
Done